template/.tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "include": ["./src/**/*"],
+  "compilerOptions": {
+    "esModuleInterop": true
+  }
+}
```

Comment:
this was the most minimal typescript config that I believe will "just work" - mostly to make jest happy.